README.md

```diff
@@ -0,0 +1,66 @@
+# **gltf-viewer**
+The GLTF Viewer is a sample application allowing for visualizing GLTF data translated from an OnShape model. It is a Node.JS application that runs as an tab inside an Onshape document. Onshape passes the document context to the viewer, which is used to help generate the GLTF visualization.
+
+This example could also be re-worked to make it a fully separate application, which would communicate with the Onshape API to obtain the document information (as opposed to Onshape providing that context for the application).
+
+## Installation
+This section outlines how to deploy and configure the application on Heroku. If you are using another service, some of these steps will not apply to you, and the equivalent steps for the other service should be taken instead.
+
+These instructions assume that the following utilities are installed: git, npm, and [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) (e.g. `brew tap heroku/brew && brew install heroku` on macOS or `sudo snap install --classic heroku` on Linux distributes that support Snap applications).
+
+1. Make a bare clone of the repository: `git clone --bare https://github.com/onshape-public/gltf-viewer.git`.
+1. Push to a new mirror repository: `cd gltf-viewer.git && git push --mirror https://github.com/youruser/my-gltf-viewer.git`.
+1. Clean up the temporary repository: `cd .. && rm -rf gltf-viewer.git`.
+1. Clone your newly mirrored repository: `https://github.com/youruser/my-gltf-viewer.git`.
+1. Create a heroku app for your project: `cd my-gltf-viewer && heroku create`. Note the URL provided in the ouput of this command.
+1. Go to the [Onshape Developer Portal](https://dev-portal.onshape.com/), create a new OAuth Application and Store entry with the following settings. Make sure that you copy the Client ID and Client Secret, as these will be needed later, and cannot be shown again.
+
+OAuth Application Setting | Value
+------------------------- | -----
+Redirect URL | https://url-from-heroku-create.herokuapp.com/oauthRedirect
+iframe URL | https://url-from-heroku-create.herokuapp.com/oauthSignin
+7. Update the `package.json` file with your new Heroku application URL:
```

Comment:
Should this be "1." to support automatic numbering?

Reply:
Goes for the rest of the README as well.

Reply:
The tables, etc. break the automatic numbering. Using 7., etc. works as expected.